modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -50,4 +58,22 @@ private function get_image_url_by_id( $id ): string {
 
 		return $image_url;
 	}
+
+	private function add_position_and_size_style( array $value ): void {
+		$default_position = self::DEFAULT_POSITION;
+
+		if ( ! isset( $value['size'] ) && ! isset( $value['position'] ) ) {
+			return;
+		}
+
+		$position = $value['position'] ?? self::DEFAULT_POSITION;
+
+		$this->style_parts[] = isset( $value['size'] )
+			? "$position / " . $value['size']
+			: $position;
+	}
+
+	private function combine_style_strings(): string {
```

Comment:
Maybe
It's more descriptive about function purpose

```suggestion
	private function generate_background_style(): string {
```